### ISSUE TITLE
plan out modernized triangle center videos

### ISSUE THREAD

**Issue by silky** [hahayes/hahayes-projects#1]
for the classic ones, see - http://vimeo.com/channels/bkcfilms/98297986

for a list of triangle centers, see - http://faculty.evansville.edu/ck6/encyclopedia/

we could pick say 4 or 5 new ones.

**Comment by silky** [comment]
"Key elements: strange fun music, circles expanding at varying speeds, nice color gradients..."

**Comment by mobeets** [comment]
http://web.archive.org/web/20090106051549/http://paideiaschool.org/TeacherPages/Steve_Sigur/resources/all%20points%20web/all-points.html#Niko

**Comment by mobeets** [comment]
![screen shot 2014-07-30 at 5 13 44 pm](https://cloud.githubusercontent.com/assets/1677179/3757268/cdd76304-1836-11e4-9f22-a14952053e50.png)

**Comment by mobeets** [comment]
a photo of our options, haha.

**Comment by mobeets** [comment]
well, the ["nine-point cirlce"](http://jwilson.coe.uga.edu/EMT668/EMT668.Folders.F97/Anderson/geometry/geometry1project/construction/construction.html) could be pretty cool

**Comment by mobeets** [comment]
[Also](http://www.jstor.org/discover/10.2307/2975188?uid=3739920&uid=2&uid=4&uid=3739256&sid=21104541242313): the Soddy line and/or Gergonne line

**Comment by silky** [comment]
i particularly like ones that lie outside the triangle.

those circle ones are also very cool i agree.

**Comment by mobeets** [comment]
really though, the cornwell triangles video is one of the most beautiful things ever. it's so hypnotizing!

anyway, seems like paper.js might be the easiest way to draw triangles, if only because [this page](http://paperjs.org/tutorials/geometry/vector-geometry/) exists.

**Comment by silky** [comment]
yeah.

we don't have to remake the videos i guess; it might be possible to just have a very simple javascript page that takes the formulas fromt the triangle center encyclopedia and just animates them nicely.

that paper.js website thing is sweet!

**Comment by mobeets** [comment]
yeah, it would be cool to have some interactivity like in Euclid: The Game where you can stretch and change the triangle to see how the center changes. but a video would be nice too.

**Comment by silky** [comment]
look out - https://scirate.com/arxiv/1408.1029 - square centers.

**Comment by mobeets** [comment]
haha, jess keeps asking me "why triangles? why not other shapes". this will explain it to her.

**Comment by silky** [comment]
a friend of mine made this!! http://pjt33.github.io/triangle-centre-fractals/tcf-canvas.html

**Comment by mobeets** [comment]
that's awesome! i don't completely understand it, but it's beautiful
